debian/install

```diff
@@ -0,0 +1 @@
+target/release/nu usr/bin
```

Comment:
This will presumably only include the `nu` binary itself but leave out all the other binaries (see: https://github.com/nushell/nushell/blob/master/Cargo.toml#L108) is that intentional?

Reply:
Good point! Thank you! I will add all of them :)